doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -0,0 +1,177 @@
+.. _how-to-enable-acrn-secure-boot-with-grub:
+
+Enable ACRN Secure Boot with GRUB
+#################################
+
+This document shows how to enable ACRN secure boot with GRUB including:
+
+  -  ACRN Secure Boot Sequence
+  -  Generate GPG Key
+  -  Setup Standalone GRUB EFI Binary
+  -  UEFI Secure Boot
+
+  Validation Environment:
+	Hardware Platform: INTEL WHL
```

Comment:
Can you change this to a pointer to the platform from the "Supported Hardware" list?

If we expect that this method can work with other platforms too, it would be good to add a note in that sense.